landoapi/spec/swagger.yml

```diff
@@ -34,7 +34,7 @@ paths:
           type: string
           description: |
             A Phabricator Conduit API key to use to get the revision.
```

Comment:
Could you add a sentence here to say that, if this value is left out, a key only capable of working with public revisions will be used instead?